ext/opcache/jit/zend_jit_arm64.dasc

```diff
@@ -11839,6 +11838,21 @@ static int zend_jit_fetch_obj(dasm_State          **Dst,
 			|	cbnz FCARG2x, >1
 			|.cold_code
 			|1:
+			|	ldr TMP1w, [FCARG2x, #offsetof(zend_property_info, flags)]
+			|	tst TMP1w, #ZEND_ACC_READONLY
+			|	bne >3
```

Comment:
``bne`` -> ``beq`` fixes ``Zend/tests/type_declarations/typed_properties_065.phpt``
I'm wonder how this worked at all.